packages/terra-i18n-plugin/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "terra-i18n-plugin",
+  "main": "lib/I18nAggregatorPlugin.js",
+  "private": true,
+  "version": "0.0.0",
+  "description": "The terra-i18n-plugin component is a webpack config plugin that examines all dependencies for translation files and then aggregates all translations for a single language into one file. Each language file is placed into an aggregated-translations directory that is then loaded by terra-i18n.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-i18n-plugin",
+    "I18n-plugin",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "devDependencies": {
+    "babel-jest": "^19.0.0",
+    "terra-toolkit": "^0.x"
+  },
+  "peerDependencies": {
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2"
+  },
+  "dependencies": {
+    "mkdirp": "^0.5.1",
+    "terra-i18n": "^0.x"
```

Comment:
I added the `i18nSuppportedLocales` file. I was a bit hesitant to make this change so would like to know others opinions on this [change](https://github.com/cerner/terra-core/pull/272/commits/ec68c739248e90704dbd53760e99077915938e57#diff-c21518f2edbdb19b046e55f1fd4d42d1R32).